### DISCUSSION TITLE
WAF Integration?

### DISCUSSION THREAD

**Discussion by zekscaler** [Automattic/jetpack#22724] [Q&A] [answered]
Hello Team;

We have some websites which is hosting service managed by Automattic. Do you have WAF integration for your services?

Thank you

**Comment by jeherve** [comment, marked as answer]
Hi there,

If your site is hosted on WordPress.com, do you think you could reach out to us [here](https://wordpress.com/help/contact), and tell us more about your needs? Once we have more information about your site and the tools you are looking for, we should be able to help!

Thank you.
